### PR TITLE
LPS-179803 As an admin user, I want to delete fields from my published object that were added before it were published

### PR DESCRIPTION
**Pull request by carolmariaabb**
**Related Issue**: https://issues.liferay.com/browse/LPS-185323

### CODE REVIEW

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java [hunk 1]
```diff
@@ -357,7 +346,12 @@ private ObjectField _toObjectField(
 				).put(
 					"update",
 					() -> {
-						if (!updateable) {
+						if ((objectDefinition.isApproved() ||
```
Comment (marcelabc) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java:349]: I did not understand why objectDefinition.isApproved() need to be in this condition. Could you explain, please?

**Review by marcelabc — APPROVED** [review]
Hi 
I just have 1 question!
I finished my review. Let me know if you have questions :smile:

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java (hunk 1 above)
Comment (carolmariaabb) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java:349]: @marcelabc I was just keeping the original behavior from [here](https://github.com/liferay-objects/liferay-portal/blob/6221e884ae62e693f52fb358482185dd16383216/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java#L323-L328). 

I believe this means that we cannot update object fields that were created before publishing the object definition.

Comment (marcelabc) [reply modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java:349]: Thanks!

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**